ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs

```diff
@@ -32,6 +32,14 @@ public class ReactTextShadowNode : LayoutShadowNode
 
         private Inline _inline;
 
+        private double? _lineHeight;
+
+        private int? _numberOfLines;
+        
+        private TextAlignment? _textAlignment;
+
+        private bool _underline;
+
         private readonly bool _isVirtual;
 
```

Comment:
Could you do me a huge favor and sort these into a logical grouping, at the very least, the readonly field should come before the mutable fields.